recipe/meta.yaml

```
@@ -22,7 +22,7 @@ source:
     folder: third_party/googletest
 
 build:
-  number: 0
+  number: 1
```

```suggestion
  number: 0
```
The build number should only be increased for a new build of the same version.

---

Thank you